src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -111,7 +111,7 @@ export class ProtractorElement implements TestElement {
     const modifierKeys = toProtractorModifierKeys(modifiers);
     const keys = rest.map(k => typeof k === 'string' ? k.split('') : [keyMap[k]])
         .reduce((arr, k) => arr.concat(k), [])
-        .map(k => Key.chord(...modifierKeys, k));
+        .map(k => modifierKeys.length > 0 ? Key.chord(...modifierKeys, k) : k);
```

Comment:
Can we have a comment here that explains why we want to avoid the chord if possible? Otherwise I can see this being refactored away.

Reply:
Good idea. Done.